examples/apps/colorapp/src/gateway/deploy.sh

```diff
@@ -11,10 +11,10 @@ fi
 # build
 docker build -t $COLOR_GATEWAY_IMAGE .
 
-# push
-if [ -z $AWS_PROFILE  ]; then
-    $(aws ecr get-login --no-include-email --region $AWS_DEFAULT_REGION)
-else
-    $(aws ecr get-login --no-include-email --region $AWS_DEFAULT_REGION --profile $AWS_PROFILE)
-fi
-docker push $COLOR_GATEWAY_IMAGE
+# # push
```

Comment:
I dont think we want to leave this commented out, right?

Reply:
oh gosh no, thanks -- totally my bad! sleep deprived, one sec!